README.md

```diff
@@ -13,7 +13,9 @@ Please note: I wasn't involved into the development of Juggluco and the final Li
 - Free Three: An offline solution -> requires root and some knowhow, also only supports libre 3 app upp to version 3.3.1. [Link (German)](https://insulinclub.de/index.php?thread/33795-free-three-ein-xposed-lsposed-modul-f%C3%BCr-libre-3-aktueller-wert-am-sperrbildschir/)
     ->Latest download: [Link](https://mega.nz/file/H51h3ILS#65mfhvDvPbtnbdWSOeXHHNxABDD60nP7iODxaDN_QPk)
 - Librelink to Nightscout sideloading: [Github repo](https://github.com/timoschlueter/nightscout-librelink-up)
-- App for downloading Linkup data and sharing with Xdrip (source: Libre 2) : [Link (German)](https://insulinclub.de/index.php?thread/32517-freestyle-libre-3/&postID=650302#post650302)
+- *LLU Clinet* App for downloading Linkup data and sharing with Xdrip (source: Libre 2) : [Link (German)](https://insulinclub.de/index.php?thread/32517-freestyle-libre-3/&postID=650302#post650302)
```

Comment:
I'm not able to merge it -> Conflicts are caused due to PR #10 . LLU client is already integrated.

Reply:
I did resolve the conflicts. Now ready for merging. Thanks you 😄